great_expectations/expectations/core/expect_column_values_to_be_decreasing.py

```diff
@@ -99,9 +100,25 @@ class ExpectColumnValuesToBeDecreasing(ColumnMapExpectation):
     }
     args_keys = ("column",)
 
+    @public_api
     def validate_configuration(
         self, configuration: Optional[ExpectationConfiguration] = None
     ) -> None:
+        """Validates the configuration of an Expectation.
+
+        The configuration will also be validated using each of the `validate_configuration` methods in its Expectation
+        superclass hierarchy.
+
+        Args:
+            configuration: An `ExpectationConfiguration` to validate. If no configuration is provided, it will be pulled
+                           from the configuration attribute of the Expectation instance.
+
+        Raises:
+            InvalidExpectationConfigurationError: The configuration does not contain the values required by the
+                                                  Expectation.
+
+            # noqa: DAR402 InvalidExpectationConfigurationError
```

Comment:
I decided to ignore the linting error since the `super` calls raise this error and if any actual logic gets added to this method it should also raise this error.